Mayden-Academy/AcademyPortal
# S1 t1 delete request fe

To test:

-Click on user in table and then delete applicant in modal popup.
-Check to to see user message of 'Applicant has been successfully deleted' on front end.
